llm_on_ray/ui/start_ui.py

```diff
@@ -1713,6 +1718,12 @@ def _init_ui(self):
         type=str,
         help="The ip:port of head node to connect when restart a worker node.",
     )
+    parser.add_argument(
+        "--ref_app_url",
+        default="http://127.0.0.1:8501",
```

Comment:
How can other external users make this work by default? can you please add document how to launch it? If not, can you please remove and add an option to make it work for you only for now?

Reply:
OK, I removed this update